### PR TITLE
Pull latest version of content loader and utils

### PR DESCRIPTION
**Pull request by bjgill**
Supplier frontend is already using v7.22.0, so it should be safe to update the frameworks repo as well.

Modify test to be able to strip out more complex paragraph tags.

### CODE REVIEW

File: tests/test_content.py [hunk 1]
```diff
@@ -107,7 +108,8 @@ def test_render_question(framework, question_id, question_set, question):
 
 
 def strip_paragraph_wrapper(value):
-    if value.startswith('<p>'):
-        value = value[3:-4]
+    match = re.fullmatch(r"^(<p.*?>)(.+?)(</p>)", value)
+    if match:
+        return match.group(2)
```
Comment (lfdebrux) [inline tests/test_content.py:113]: Could you add a commit message explaining why this change is needed?

**Review by lfdebrux — APPROVED** [review]
Great! ➕

File: tests/test_content.py (hunk 1 above)
Comment (bjgill) [reply tests/test_content.py:113]: :+1: 

    Modify test to be able to strip out more complex paragraph tags. We need to do this because the previous logic was only able to strip '<p>'. However, as of https://github.com/alphagov/digitalmarketplace-content-loader/pull/77, we get more complex paragraph tags such as '<p class="govuk-body">'. The new regex should be able to detect these more complex paragraph tags and allow us to strip them.

**Comment by bjgill** [comment]
Rebased (with better commit message) and bumped minor version. Just awaiting Travis now
